docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,228 @@
+{ml} processes (these processes are separate to {es} JVM) via the 
+`xpack.ml.max_machine_memory_percent` setting. If the node is a dedicated {ml} 
+node, increase this value from the default 30%. **If you have a cloud 
+deployment, you don't need to manually increase this value, it is automatically 
+set.**
+
+
+[[max-model-memory-limit]]
+=== Setting model memory limit properly
+
+There are two model memory limit configuration options:
+
+* `xpack.ml.max_model_memory_limit` is a node setting that defines the maximum 
+  `model_memory_limit` value you can set for any {anomaly-job} on the node. If 
+  the memory requirement of a job is higher than this value, an error occurs 
+  and the job cannot be created.
+
+* `model_memory_limit` is a job variable that sets the approximate maximum 
+  amount of memory resources required for analytical processing. If this 
+  variable is set too low for the job and the limit is approached, data 
+  discarding becomes more aggressive. Upon exceeding this limit, new entities 
+  are not modeled.
+
+Use model memory estimation to have a better picture of the memory needs of the 
+model. Model memory estimation happens automatically when you create the job in 
+{kib} or you can call the {ref}//ml-estimate-model-memory.html[Estimate 
+{anomaly-jobs} model memory API] manually. The estimation is based on the 
+analysis configuration details for the job and cardinality estimates for the 
+fields it references.
+
+
+[[job-config]]
+== Job configuration guidelines
+
+
+[[dedicated-results-index]]
+=== Using a dedicated results index
+
+If a {anomaly-job} generates a lot of data, use a dedicated index to store the 
+results. This way, you can retain results in a shorter time. If you use a shared 
+index, the different {anomaly-jobs} may slow down.
+
+When you want to delete a job, the delete job API instantly drops the index 
+containing the job’s results if you use a dedicated index. If you use a shared 
+one, however, it may take hours to delete all the individual documents related 
+to the deleted job.
+
+
+[[search-optimization]]
+=== Optimizing search queries
+
+There are different ways to write {es} queries and some of them are more 
+efficient than others. If you are operating on a big scale, make sure that your 
+{dfeed} query is as efficient as possible. Consult 
+{ref}/tune-for-search-speed.html[Tune for search speed] to learn more about {es} 
+performance tuning.
+
+
+[[influencers-detectors]]
+=== Choosing influencer fields and detectors
+
+Optimize your {anomaly-job} by choosing only relevant influencer fields and 
+detectors.
+
+
+[[short-bucket-span]]
+=== Avoiding short bucket span for large jobs
+
+If you have a large amount of data, use longer bucket span. If your data 
+requires short bucket span, split the data to avoid having high cardinality. The 
+combination of short bucket span and high cardinality is a resource-intensive 
+situation that you may want to avoid.
```

Comment:
Can you try to work some of this into the section... 


For machine data, we typically see bucket spans of between 15m to 1h, but this depends on the 
data and the use case and the latency required for alerting. By using longer bucket spans, less
resources are required for processing each bucket and fewer results are written. 
For example, doubling the bucket span will halve the amount of work done
and may still give meaningful results.

Please note that bucket spans that are sensible dividers of an hour / day work best, as most periodic patterns have a daily cycle. e.g. 1m, 5m, 10m, 15m, 30m, 1h